packages/cli/src/commands/process/dev.ts

```diff
@@ -1,62 +1,91 @@
-import {IProcess} from '@mesg/api/lib/process'
-import {hash} from '@mesg/api/lib/types'
+import { flags, Command } from '@oclif/command'
+import Listr from 'listr'
+import * as Environment from '../../tasks/environment'
+import * as Process from '../../tasks/process'
+import * as Execution from '../../tasks/execution'
+import version from '../../version'
+import API from '@mesg/api'
+import LCDClient from '@mesg/api/lib/lcd'
+import { join } from 'path'
 import * as base58 from '@mesg/api/lib/util/base58'
+import chalk from 'chalk'
+import { inspect } from 'util'
+import { decode } from '@mesg/api/lib/util/encoder'
 
-import Command from '../../root-command'
-import {IsAlreadyExistsError} from '../../utils/error'
+const ipfsClient = require('ipfs-http-client')
 
-import ProcessCompile from './compile'
-import ProcessCreate from './create'
-import ProcessDelete from './delete'
-import ProcessLog from './logs'
-
-export default class ProcessDev extends Command {
-  static description = 'Run a process in development mode'
+export default class Dev extends Command {
+  static description = 'Compile a process'
 
   static flags = {
-    ...Command.flags,
-    ...ProcessCompile.flags,
-    ...ProcessCreate.flags,
+    image: flags.string({ name: 'MESG engine image', default: 'mesg/engine' }),
+    tag: flags.string({ name: 'MESG engine version', default: version.engine }),
+    pull: flags.boolean({ name: 'Force to pull the docker image', default: false }),
+    configDir: flags.string({ name: 'Directory for your configurations', default: join(process.cwd(), '.mesg') }),
```

Comment:
The volume needs to be mounted because the config file contains the mnemonic for the engine.

For the location, I would like to avoid to put it in the `~/.mesg` as I had so many times issues with that because I was running the testnet on the `~/.mesg` and override it many times in development.

Maybe the project base is too much, but at least the cli config path to really see this env as just an env for the CLI and nothing else.